extensions/BMO/template/en/default/bug/create/create-firefox-security.html.tmpl

```diff
@@ -0,0 +1,188 @@
+  if (!isFilledOut('comment')) alert_text += 'Please enter a value for comment.\n';
+  if (alert_text != '') {
+    alert(alert_text);
+    return false;
+  }
+  return true;
+}
+[% END %]
+
+[% PROCESS global/header.html.tmpl
+   title = "Firefox Security Form"
+   generate_api_token = 1
+   style = inline_style
+   javascript = inline_javascript
+   javascript_urls = [ 'extensions/BMO/web/js/form_validate.js',
+                       'extensions/BMO/web/js/attach-desc.js',
+                       'js/field.js', 'js/util.js' ]
+%]
+
+[% USE Bugzilla %]
+[% cgi = Bugzilla.cgi %]
+
+<h1>Firefox Security Form</h1>
+
+<form id="firefox_security_form" method="post" action="post_bug.cgi" enctype="multipart/form-data"
+      onSubmit="return validateAndSubmit();">
+  <input type="hidden" name="product" value="Firefox">
+  <input type="hidden" name="component" value="Security">
+  <input type="hidden" name="rep_platform" value="unspecified">
+  <input type="hidden" name="op_sys" value="unspecified">
+  <input type="hidden" name="version" value="unspecified">
+  <input type="hidden" name="bug_severity" id="bug_severity" value="normal">
+  <input type="hidden" name="priority" id="priority" value="--">
+  <input type="hidden" name="target_milestone" id="target_milestone" value="---">
+  <input type="hidden" name="status_whiteboard" id="status_whiteboard"
+         value="[reporter-external] [web-bounty-form] [verif?]">
+  <input type="hidden" name="bug_severity" id="bug_severity" value="normal">
+  <input type="hidden" name="groups" value="firefox-core-security">
+  <input type="hidden" name="flag_type-803" id="flag_type-803" value="?">
+  <input type="hidden" name="token" value="[% token FILTER html %]">
+
+<div class="head_desc">
+  <a href="https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines?redirectlocale=en-US&redirectslug=Bug_writing_guidelines">
+    [% terms.Bug %] writing guidelines</a>
+</div>
+
+<p>
+  [ <span class="required_star">*</span> <span class="required_explanation">Required Field</span> ]
+</p>
+
+
+<div class="form_section">
+  <label for="short_desc" class="field_label required">Summary</label>
+  <div class="field_desc">
+    A short description of the issue being reported including the version of
+    Firefox and operating system used.
+  </div>
+  <input required type="text" name="short_desc" id="short_desc" size="80">
+</div>
+
+<div class="form_section">
+  <label for="comment" class="field_label required">Description</label>
+  <div class="field_desc">
+    How was this issue discovered, include the steps, tools or other information that
+    will help reproduce and diagnose the issue. A good primer on what to include can
+    be found <a href="https://developer.mozilla.org/en-US/docs/Mozilla/QA">here</a>.
+  </div>
+  <textarea required id="comment" name="comment" cols="80" rows="5"></textarea>
+</div>
+
+<div class="form_section">
```

Comment:
We don't need URL section for client bounty reports - this section can be reviewed.